circle.yml

```diff
@@ -7,6 +7,15 @@ test:
     - gulp -v
   override:
     - npm test # Run the package and docs test suite
-    - npm run build:package # Run the release process
+    - npm run build
   post:
-    - ls -agolf dist/ # Ensure that build:package worked
\ No newline at end of file
+    - ls -agolf dist/ # Ensure that build:package worked
+
+deployment:
+  npm:
+    branch: master
+    owner: 18F
+    commands:
+      - npm run build:package # Build distributed package files
+      - echo -e "$NPM_USERNAME\n$NPM_PASSWORD\n$NPM_EMAIL" | npm login
+      - npm run publish
```

Comment:
We had good success using a npm module called `check-publish` for this, as it will check if the current package.json version is already on npm, and only attempt to publish if it's not.

See [cg-style build](https://github.com/18F/cg-style/blob/master/circle.yml#L33)